packages/spec/integration/typescript/package.json

```diff
@@ -9,11 +9,13 @@
     "testEnvironment": "../../helpers/env.js"
   },
   "scripts": {
-    "start": "node -e 'require(\"hops\").run(\"start\")'"
+    "start": "node -e 'require(\"hops\").run(\"start\")'",
+    "tsc": "tsc"
```

Comment:
I think renaming the script to `type-check` makes more sense.

Reply:
Fine with me, will adjust it in Readme as well then